db/data/20191022155610_convert_qts_award_year.rb

```diff
@@ -0,0 +1,11 @@
+class ConvertQtsAwardYear < ActiveRecord::Migration[5.2]
```

Comment:
I know it's simple, but can we get a test for this? There's some prior art in #530 for how we did this. I was burnt by a data migration without tests before

Reply:
Ah yes, I figured it was such a simple query that we'd just be testing ActiveRecord. I'll add one 👍